configuration/lambda_config.template

```diff
@@ -26,7 +26,6 @@
     "disable_existing_loggers": "False",
     "formatters": {
       "default": {
-        "format": "$EXODUS_LOG_FORMAT",
```

Comment:
The other solution would be to make sure the format string is quoted appropriately.  Really, the problem we hit is not specific to usage of JSON - attempting to use any log format containing a `"` character or various other characters would have failed for the same reason.

That being said, I like the simplicity of the current solution.

Reply:
Yeah, it'sas tough with bash and JSON's different issues surrounding single and double quotes.
`{'item': 'value'}` may have worked in bash, for instance, but JSON won't accept it.

I think that the need for JSON formatted logs for splunk helps further justify removing log format configuration too.